src/ui/server/api/keystone/account/index.ts

```diff
@@ -33,6 +35,35 @@ const account = {
       throw new Error('Creating exporter account');
     }
   },
+  get: async (id: string) => {
+    try {
+      console.info('Getting exporter account');
+
+      const variables = { id };
+
+      const response = (await apollo('POST', getExporterQuery, variables)) as ApolloResponse;
+
+      if (response.errors) {
+        console.error('GraphQL error getting exporter account ', response.errors);
+      }
+
+      if (response?.networkError?.result?.errors) {
+        console.error('GraphQL network error getting exporter account ', response.networkError.result.errors);
+      }
+
+      if (response?.data?.exporter) {
+        const { data } = response;
```

Comment:
```
const { data: { exporter} } = response;
return exporter;
```

or we return straight away the exporter?
```
return response.data.exporter;
```

Reply:
Great shout @IgnatG i've updated all instances of this in the UI API calls 👍